examples/pod_failure_kill.yaml

```diff
@@ -0,0 +1,16 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+apiVersion: chaos.datadoghq.com/v1beta1
+kind: Disruption
+metadata:
+  name: pod-failure-kill
+  namespace: chaos-demo
+spec:
+  selector:
+    app: demo-curl
+  count: 1
+  podFailure:
+    kill: true # send a SIGTERM signal to the pod's containers
```

Comment:
Just confirming, this comment is wrong, yes? `kill: true` means we're sending a `SIGKILL`, not a `SIGTERM`

Reply:
Yes, this is wrong. I'll push a couple of changes and fixes including this one.